### PR TITLE
fix: Fix showing analytics-opt-out in config as missing

### PR DESCRIPTION
**Pull request by dbolson**
**Requirements**

- [ ] I have added test coverage for new or changed functionality
- [ ] I have followed the repository's [pull request submission guidelines](../blob/main/CONTRIBUTING.md#submitting-pull-requests)
- [ ] I have validated my changes against all supported platform versions

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**

Provide a clear and concise description of what you expect to happen.

**Describe alternatives you've considered**

Provide a clear and concise description of any alternative solutions or features you've considered.

**Additional context**

Add any other context about the pull request here.

### CODE REVIEW

File: cmd/resources/resources.go [hunk 1]
```diff
@@ -137,7 +137,7 @@ func NewOperationCmd(parentCmd *cobra.Command, client *http.Client, op Operation
 	}
 
 	opCmd.cmd = cmd
-	opCmd.initFlags()
+	_ = opCmd.initFlags()
```
Comment (dbolson) [inline cmd/resources/resources.go:140]: Fixes lint error `Error return value of `opCmd.initFlags` is not checked (errcheck)`

File: cmd/config/config.go [hunk 2]
```diff
@@ -46,8 +46,6 @@ func NewConfigCmd(analyticsTracker analytics.Tracker) *cobra.Command {
 	cmd.Flags().String(UnsetFlag, "", "Unset a config field")
 	_ = viper.BindPFlag(UnsetFlag, cmd.Flags().Lookup(UnsetFlag))
 
-	// TODO: running config should show help
```
Comment (dbolson) [inline cmd/config/config.go:49]: This was fixed.

File: internal/output/plaintext_fns.go [hunk 3]
```diff
@@ -17,7 +17,7 @@ var ConfigPlaintextOutputFn = func(r resource) string {
 
 	lst := make([]string, 0)
 	for _, k := range keys {
-		lst = append(lst, fmt.Sprintf("%s: %s", k, r[k]))
+		lst = append(lst, fmt.Sprintf("%s: %v", k, r[k]))
```
Comment (dbolson) [inline internal/output/plaintext_fns.go:20]: Without this change, running `config --list` would show `analytics-opt-out: %!!(MISSING)s(bool=false)`.

File: internal/config/config.go [hunk 4]
```diff
@@ -14,7 +14,7 @@ const Filename = ".ldcli-config.yml"
 // ConfigFile represents the data stored in the config file.
 type ConfigFile struct {
 	AccessToken     string `json:"access-token,omitempty" yaml:"access-token,omitempty"`
-	AnalyticsOptOut bool   `json:"analytics-opt-out,omitempty" yaml:"analytics-opt-out,omitempty"`
+	AnalyticsOptOut *bool  `json:"analytics-opt-out,omitempty" yaml:"analytics-opt-out,omitempty"`
```
Comment (dbolson) [inline internal/config/config.go:17]: The `omitempty` would kick in without this, so a `false` value would not get displayed with `config --list`.

Comment (k3llymariee) [reply internal/config/config.go:17]: ooops thanks for catching

**Review by k3llymariee — APPROVED** [review]

File: internal/config/config.go (hunk 4 above)
Comment (dbolson) [reply internal/config/config.go:17]: We _should_ have tests that write the config to an in-memory file system and read it back to catch stuff like this, but I didn't spend the effort on it.

**Pull request merged**